x-pack/test/functional/apps/search_playground/playground_overview.ess.ts

```diff
@@ -18,76 +24,144 @@ export default function (ftrContext: FtrProviderContext) {
+          await pageObjects.searchPlayground.PlaygroundStartChatPage.expectAddedConnectorCallout();
+        });
       });
 
-      it('click add connector button opens connector flyout', async () => {
-        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectOpenConnectorPagePlayground();
-      });
+      describe('without any indices', () => {
+        it('show no index callout', async () => {
+          await pageObjects.searchPlayground.PlaygroundStartChatPage.expectNoIndexCalloutExists();
+          await pageObjects.searchPlayground.PlaygroundStartChatPage.expectCreateIndexButtonToExists();
+        });
 
-      it('hide gen ai panel when connector exists', async () => {
-        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectHideGenAIPanelConnector(
-          createConnector
-        );
+        it('hide no index callout when index added', async () => {
+          await createIndex();
+          await pageObjects.searchPlayground.PlaygroundStartChatPage.expectSelectIndex(indexName);
+        });
       });
 
-      it('show chat page', async () => {
-        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectSelectIndex(indexName);
-        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectToStartChatPage();
+      describe('with existing indices', () => {
+        it('dropdown shows up', async () => {
+          await pageObjects.searchPlayground.PlaygroundStartChatPage.expectIndicesInDropdown();
+        });
+
+        it('can select index from dropdown and navigate to chat window', async () => {
+          await pageObjects.searchPlayground.PlaygroundStartChatPage.expectToSelectIndicesAndStartButtonEnabled(
+            indexName
+          );
+        });
       });
     });
 
     describe('chat page', () => {
-      it('chat works', async () => {
-        await pageObjects.searchPlayground.PlaygroundChatPage.expectChatWorks();
+      it('loads successfully', async () => {
+        await pageObjects.searchPlayground.PlaygroundChatPage.expectChatWindowLoaded();
       });
 
-      it('open view code', async () => {
-        await pageObjects.searchPlayground.PlaygroundChatPage.expectOpenViewCode();
-      });
+      describe('chat', () => {
+        it('works', async () => {
+          const conversationInterceptor = proxy.intercept(
+            'conversation',
+            (body) =>
+              (JSON.parse(body) as OpenAI.Chat.ChatCompletionCreateParamsNonStreaming).tools?.find(
+                (fn) => fn.function.name === 'title_conversation'
+              ) === undefined
+          );
 
-      it('show fields and code in view query', async () => {
-        await pageObjects.searchPlayground.PlaygroundChatPage.expectViewQueryHasFields();
-      });
+          await pageObjects.searchPlayground.PlaygroundChatPage.sendQuestion();
+
+          const [conversationSimulator] = await Promise.all([
+            conversationInterceptor.waitForIntercept(),
+          ]);
```

Comment:
we don't need here promise.all. guess just forgot to remove it after reducing titleInterceptor